docs/source/install.rst

```diff
@@ -84,6 +85,96 @@ Raspberry Pi OS
     sudo curl -fL http://docs.luxonis.com/_static/install_dependencies.sh | bash
 
 
+Jetson Nano
+***********
+
+To install DepthAI on Jetson Nano, perform the following steps, after completing a fresh install and setup. On the first log in, 
+**do not** immediately run updates.
+
+Go to the *Software* (App Store) and delete the apps or software that you probably will not use. Then open a terminal window and run the following commands:
```

Comment:
I don't have "Software" or App Store on my jetson
![image](https://user-images.githubusercontent.com/5244214/117980289-1b24cd00-b334-11eb-8be6-d9a109947ff6.png)

Reply:
yes, you have :)